packages/zoe/src/contractSupport/ratio.js

```diff
@@ -352,6 +352,10 @@ export const ratiosSame = (left, right) => {
 export const quantize = (ratio, newDen) => {
   const oldDen = ratio.denominator.value;
   const oldNum = ratio.numerator.value;
+  if (oldDen < newDen) {
```

Comment:
ignorable nit: testing the param against the status quo reads more conventionally to me
```suggestion
  if (newDen > oldDen) {
```

Reply:
very correct.